### PR TITLE
split std.algorithm unittest compilation

### PR DESCRIPTION
**Pull request by MartinNowak**
- triggers OOM with ddmd

### CODE REVIEW

**Comment by 9rnsr** [comment]
Auto-merge toggled on

**Comment by quickfur** [comment]
Why not split it into per-submodule unittesting instead? (i.e., test `std.algorithm.iteration` on its own, then `std.algorithm.searching`, etc.).

File: win64.mak [hunk 1]
```diff
@@ -138,16 +138,20 @@ SRC_STD_CONTAINER= std\container\array.d std\container\binaryheap.d \
...
 
-SRC_STD_ALGO= std\algorithm\package.d std\algorithm\comparison.d \
-	std\algorithm\iteration.d std\algorithm\mutation.d \
-	std\algorithm\searching.d std\algorithm\setops.d \
+SRC_STD_ALGO_1=std\algorithm\package.d std\algorithm\comparison.d \
+	std\algorithm\iteration.d std\algorithm\mutation.d
+SRC_STD_ALGO_2=std\algorithm\searching.d std\algorithm\setops.d \
 	std\algorithm\sorting.d std\algorithm\internal.d
+SRC_STD_ALGO=$(SRC_STD_ALGO_1) $(SRC_STD_ALGO_2)
+std\algorithm\package.d std\algorithm\comparison.d \
```
Comment (yebblies) [inline win64.mak]: Should be deleted?

**Comment by yebblies** [comment]
> Why not split it into per-submodule unittesting instead?

We sometimes catch bugs this way that don't show up that way.

**Pull request merged**